recipes/careamics/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "careamics" %}
+{% set version = "0.0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/careamics-{{ version }}.tar.gz
+  sha256: a1261fa0d552fd0d8fe0a956c64525957fd80fdd0906ef112028f9eeefaf030e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python {{ python_min }}
+    - hatchling
+    - hatch-vcs
+  run:
+    - python >={{ python_min }}
+    - numpy <2.0.0
+    - pytorch >=2.0.0
+    - torchvision
+    - tifffile
+    - psutil
+    - pydantic >=2.5,<2.9
+    - pytorch-lightning >=2.2.0
+    - pyyaml
+    - typer ==0.12.3
+    - scikit-image <=0.23.2
+    - zarr <3.0.0
+    - bioimageio.core >=0.6.9
+
+test:
+  imports:
+    - careamics
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://careamics.github.io
+  summary: Deep-learning library for running N2V and friends.
+  description: |
+    CAREamics is a PyTorch library aimed at simplifying the use of Noise2Void and its many 
+    variants and cousins (CARE, Noise2Noise, N2V2, P(P)N2V, HDN, MicroSplit etc.).
+
+    Documentation
+    -------------
+    - https://careamics.github.io
```

Comment:
You have another URL for docs below. Let's make the description easier to ready.

```suggestion
```